src/connectors/answers/connectAnswers.ts

```diff
@@ -0,0 +1,179 @@
+import {
+  checkRendering,
+  createDocumentationMessageGenerator,
+  createConcurrentSafePromise,
+  debounce,
+  addAbsolutePosition,
+  noop,
+} from '../../lib/utils';
+import { Connector, Hits, Hit, FindAnswersResponse } from '../../types';
+
+const withUsage = createDocumentationMessageGenerator({
+  name: 'answers',
+  connector: true,
+});
+
+export type AnswersRendererOptions = {
+  /**
+   * The matched hits from Algolia API.
+   */
+  hits: Hits;
+
+  /**
+   * whether it's still loading the results from the /answers API
+   */
+  isLoading: boolean;
+};
+
+export type AnswersConnectorParams = {
+  /**
+   * Attributes to use for predictions.
+   * If empty, we use all `searchableAttributes` to find answers.
+   * All your `attributesForPrediction` must be part of your `searchableAttributes`.
+   */
+  attributesForPrediction?: string[];
+
+  /**
+   * The languages in the query. Currently only supports `en`.
+   */
+  queryLanguages: string[];
```

Comment:
```suggestion
  queryLanguages: Array<'en'>;
```

maybe this is good for the InstantSearch widget?